modules/cnf-topology-aware-lifecycle-operator-apply-policies.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-apply-policies_{context}"]
+= Applying update policies to managed clusters
+
+The Topology Aware Lifecycle Operator makes enforced copies of the managed {rh-rhacm} policies. Each copied policy has its own corresponding {rh-rhacm} placement rule and {rh-rhacm} placement binding. As the remediation plan progresses, the clusters are added one by one in the placement rule that corresponds with the applicable managed policy to the current cluster in the current batch. After a batch is updated, all clusters are removed from the placement rules associated with the copied policies. Then, the update of the next batch starts.
+
+.Prerequisites
+
+* Install the Topology Aware Lifecycle Operator.
+* Provision one or more managed SNO clusters using ZTP.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the `ClusterGroupUpgrade` CR, and then save the YAML in the `cgu-cr.yaml` file:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: du-upgrade-4915
+  namespace: ztp-group-du-sno
+spec:
+  deleteObjectsOnCompletion: true
```

Comment:
I think deleteObjectsOnCompletion is deprecated (see https://bugzilla.redhat.com/show_bug.cgi?id=2040874). Now it should be as below (under spec):

```
spec:
  actions:
    afterCompletion:
      deleteObjects: true
```